projects/vdk-plugins/airflow-provider-vdk/vdk_provider/hooks/vdk.py

```diff
@@ -44,7 +46,7 @@ def __init__(
         conn_id: str,
         job_name: str,
         team_name: str,
-        timeout: int = 1,  # TODO: Set reasonable default
+        timeout: int = 5,  # TODO: Set reasonable default
```

Comment:
Why the change?

Reply:
It seems the hook connection times out too easily, at least that's my impression from testing it.